core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.html

```diff
@@ -23,6 +23,9 @@ <h3>Upload Thumbnail</h3>
       <div class="oppia-form-error" *ngIf="invalidImageWarningIsShown">
         Please upload a valid SVG image.
       </div>
+      <div class="oppia-form-error" *ngIf="invalidFilenameWarningIsShown">
+        Please upload a valid filename image.
+      </div>
```

Comment:
Can we make this more descriptive? We should tell the user what we do not allow.